<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upTSClass</class>
 <widget class="QMainWindow" name="upTSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Teachers or Subjects</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/imgs/images/window_icon.ico</normaloff>:/imgs/images/window_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnTeachName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnTeachID">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>52</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnTeachSurname">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTeacherUpdate">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>390</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Teacher Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Teacher Surname</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnSubID">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>52</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubUpdate">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnSubName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbnTeacher">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblQueryStatus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tbSubjects">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>431</width>
      <height>234</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tbTeachers">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>spnSubID</tabstop>
  <tabstop>lnSubName</tabstop>
  <tabstop>cbnTeacher</tabstop>
  <tabstop>btnSubUpdate</tabstop>
  <tabstop>spnTeachID</tabstop>
  <tabstop>lnTeachName</tabstop>
  <tabstop>lnTeachSurname</tabstop>
  <tabstop>btnTeacherUpdate</tabstop>
  <tabstop>tbSubjects</tabstop>
  <tabstop>tbTeachers</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
